hazelcast-sql/src/test/java/com/hazelcast/sql/impl/plan/cache/PlanCacheIntegrationTest.java

```diff
@@ -45,7 +45,7 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class PlanCacheIntegrationTest extends PlanCacheTestSupport {
 
-    private final TestHazelcastInstanceFactory factory = new TestHazelcastInstanceFactory(2);
+    private final SqlTestInstanceFactory factory = SqlTestInstanceFactory.create();
```

Comment:
Plan cache tests are not related to expressions. Do you really want to run it in Jet environment?

Reply:
It should not hurt...

Reply:
Running the tests is not about expressions, but about jet modifying the optimization.